src/node_file.cc

```diff
@@ -1150,28 +1153,133 @@ static void RMDir(const FunctionCallbackInfo<Value>& args) {
   }
 }
 
+int MKDirpSync(uv_loop_t* loop, uv_fs_t* req, const std::string& path, int mode,
+               uv_fs_cb cb = nullptr) {
+  FSContinuationData continuation_data = FSContinuationData(req, mode, cb);
```

Comment:
I think the compiler could optimize the copy operation away here, but either way, it’s a bit more idiomatic to initialize this as `FSContinuationData continuation_data(req, mode, cb);` (i.e. without the assignment) :slightly_smiling_face: